### PR TITLE
docs: fixes breakage from confluent-hub

### PR DESCRIPTION
**Pull request by MichaelDrogalis**
### Description 

Fixes issues with using embedded confluent-hub after 0.9.0's change to UBI.

### CODE REVIEW

File: docs/tutorials/etl.md [hunk 1]
```diff
@@ -31,24 +31,30 @@ This tutorial shows how to create a streaming ETL pipeline that ingests and join
 
 ### Get the connectors
 
-To get started, download the connectors for Postgres, MongoDB, and Elasticsearch to a fresh directory. You can either get them by using [confluent-hub](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html), or by running the following one-off Docker commands that wrap it.
+To get started, download the connectors for Postgres, MongoDB, and Elasticsearch to a fresh directory. The easiest way to do that is by using [confluent-hub](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html).
```
Comment (JimGalasyn) [inline docs/tutorials/etl.md:34]:
```suggestion
To get started, download the connectors for Postgres, MongoDB, and Elasticsearch to a fresh directory. The easiest way to do this is by using [confluent-hub](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html).
```

File: docs/tutorials/materialized.md [hunk 2]
```diff
@@ -27,13 +27,21 @@ This tutorial shows how to create and query a set of materialized views about ph
 
 ### Get the Debezium connector
 
-To get started, download the Debezium connector to a fresh directory. You can either get that using [confluent-hub](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html), or by running the following one-off Docker command that wraps it:
+To get started, download the Debezium connector to a fresh directory. The easiest way to do that is by using [confluent-hub](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html).
```
Comment (JimGalasyn) [inline docs/tutorials/materialized.md:30]:
```suggestion
To get started, download the Debezium connector to a fresh directory. The easiest way to do this is by using [confluent-hub](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html).
```

**Comment by colinhicks** [comment]
Probably 100% clear from the description, but we'll want to cherry pick this into `0.9.0-ksqldb` only, as the other versions use the Debian base image.

**Comment by MichaelDrogalis** [comment]
@JimGalasyn Suggestions applied.

**Comment by JimGalasyn** [comment]
@colinhicks Cherry-picks to `0.9.0-ksqldb` and `0.9.x-ksqldb` are done!

**Pull request merged**